redaxo/src/addons/metainfo/lib/handler/handler.php

```diff
@@ -686,8 +686,9 @@ protected function fireCallbacks(rex_sql $sqlFields)
                     $fieldType = $field->getValue('type_id');
                     $fieldAttributes = $field->getValue('attributes');
                     $fieldValue = self::getSaveValue($fieldName, $fieldType, $fieldAttributes);
+                    $fieldId = (int) $field->getValue('id');
```

Comment:
ich bin am überlegen, ob wir in rex_sql eine `getIntValue`-Methode aufnehmen sollten, da man ja sehr oft ints abfragt.

Reply:
oder in `getValue` nen 2. parameter um den typ zu übergeben (analog rex_get() etc)